src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const STATES = [];
+  let copy = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear':
+        copy = {};
+        break;
+      case 'addProperties':
```

Comment:
Add the blank line between the cases
```suggestion
        break;
        
      case 'addProperties':
```